Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_description.md

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
We have nothing in the detailed description?

Reply:
Not from the previous integration, but i'll add something. the access to tables was a reoccurring issue on slack conversations.